### PR TITLE
Analytics updates

### PR DESCRIPTION
**Pull request by olivierapivideo**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208145821422434

### CODE REVIEW

File: oas_apivideo.yaml [hunk 1]
```diff
@@ -13088,7 +13097,11 @@ paths:
       - name: viewDuration
         in: query
         description: | 
-          TODO
+          Use this query parameter to define how many seconds a view has to last to be counted in analytics data.
+
+          - You can only use this parameter together with the `view` metric.
```
Comment (mioara-api-video) [inline oas_apivideo.yaml:13102]: I'd remove the with maybe?
"You can only use this parameter with the `view` metric."
Or maybe "You can only use this parameter along the `view` metric." ?

(i'm forgetting how to English)

File: oas_apivideo.yaml [hunk 2]
```diff
@@ -13335,7 +13365,7 @@ paths:
           - `device-type`: Returns analytics based on the type of device used by the viewers. Response values can include: `computer`, `phone`, `tablet`, `tv`, `console`, `wearable`, `unknown`.
           - `operating-system`: Returns analytics based on the operating system used by the viewers. Response values can include `windows`, `mac osx`, `android`, `ios`, `linux`.
           - `browser`: Returns analytics based on the browser used by the viewers. Response values can include `chrome`, `firefox`, `edge`, `opera`.
-          - `referrer`: TODO
+          - `referrer`: Returns the URL where the view originates from, for example a website where the video is embedded. View events from Android and iOS return empty strings as referrer.
```
Comment (mioara-api-video) [inline oas_apivideo.yaml:13368]: Last sentence should maybe be: "View events from Android and iOS return empty strings as the [or a] referrer."

File: oas_apivideo.yaml [hunk 3]
```diff
@@ -13415,7 +13445,16 @@ paths:
       - name: unique
         in: query
         description: |
-          TODO
+          Use this query parameter to control how viewer data is counted:
+
+          - `true` means that a single user watchig multiple times counts as 1 unique viewer
```
Comment (mioara-api-video) [inline oas_apivideo.yaml:13450]: watching

File: oas_apivideo.yaml [hunk 4]
```diff
@@ -13079,7 +13079,16 @@ paths:
...
```
Comment (mioara-api-video) [inline oas_apivideo.yaml:13084]: watching

File: oas_apivideo.yaml (hunk 3 above)
Comment (szekelyzol) [reply oas_apivideo.yaml:13450]: Nice catch!

File: oas_apivideo.yaml [hunk 5]
```diff
@@ -13079,7 +13079,16 @@ paths:
...
+          - `false` means that all views count, even if from the same user.
+
+          The API accepts this parameter only when you use the `ccv-peak`, `ccv-average`, or `view` metric.
```
[the remark this answers is not in the document]
Comment (szekelyzol) [reply oas_apivideo.yaml:13087]: Nice catch, fixed! For this route, the sentence should be:

> The API accepts this parameter only when you use the `ccv` or `view` metric.

File: oas_apivideo.yaml (hunk 1 above)
Comment (szekelyzol) [reply oas_apivideo.yaml:13102]: Fixed: removed "together".

File: oas_apivideo.yaml (hunk 3 above)
Comment (szekelyzol) [reply oas_apivideo.yaml:13450]: Fixed ✅

File: oas_apivideo.yaml (hunk 4 above)
Comment (szekelyzol) [reply oas_apivideo.yaml:13084]: Fixed ✅

File: oas_apivideo.yaml [hunk 6]
```diff
@@ -13701,11 +13763,10 @@ paths:
       - name: interval
         in: query
         description: |
-          Use this query parameter to define how granularity of the data. Possible values: `minute`, `hour`, `day`.
-
-          - Default: If no interval specified and the period (different between from and to) ≤ 2 days then hour, otherwise day.
+          Use this query parameter to define the granularity of the data. Possible values: `minute`, `hour`, `day`.
 
           - If you do not set a value for `interval`, and the period you set using the `from` and `to` parameters is less than or equals to 2 days, then the default assigned value is `hour`. Otherwise the API sets it to `day`.
+          - When you set `minute` as interval, the timeframe you define with the `from` and `to` parameters should be less than 60 minutes.
```
[the remark this answers is not in the document]
Comment (szekelyzol) [reply oas_apivideo.yaml:13769]: Corrected to `must`

**Review by mioara-api-video — APPROVED** [review]

**Pull request merged**